src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,825 @@
+;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+; COOKIE: 6ff7ea72c14ce2d4 (echoed)
+;; QUESTION SECTION:
+;nginx.nginx-test.svc.clusterset.local. IN	A
+
+;; ANSWER SECTION:
+nginx.nginx-test.svc.clusterset.local. 5 IN A	100.3.220.176
+
+;; Query time: 16 msec
+;; SERVER: 100.2.0.10#53(100.2.0.10)
+;; WHEN: Mon Nov 30 17:52:55 UTC 2020
+;; MSG SIZE  rcvd: 125
+```
+<!-- markdownlint-enable no-hard-tabs -->
+Note that DNS resolution works across the clusters, and that the IP address **100.3.220.176** returned is the same ClusterIP associated with
+the `nginx` Service on **cluster3**.
+
+##### 4. Create an `nginx` Deployment on **cluster2**
+
+If multiple clusters export a Service with the same name and from the same namespace, it will be recognized as a single combined Service.
+To test this, let's deploy the same `nginx` Service, this time on the nginx-test namespace on **cluster2**:
+
+```bash
+$ kubectl -n nginx-test create deployment nginx --image=nginxinc/nginx-unprivileged:stable-alpine
+deployment.apps/nginx created
+
+$ kubectl -n nginx-test expose deployment nginx --port=8080
+service/nginx exposed
+```
+
+This will create an `nginx` Service on the nginx-test namespace which targets TCP port 8080 on any Pod with the `app: nginx` label, and
+expose it on an abstracted Service port. When created, the Service is assigned a unique IP address (also called ClusterIP):
+
+```bash
+$ kubectl -n nginx-test get service nginx
+NAME    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+nginx   ClusterIP   100.2.29.136   <none>        8080/TCP   1m40s
+```
+
+```bash
+$ kubectl -n nginx-test get pods -l app=nginx -o wide
+NAME                     READY   STATUS    RESTARTS   AGE   IP           NODE              NOMINATED NODE   READINESS GATES
+nginx-5578584966-d7sj7   1/1     Running   0          22s   10.2.224.3   cluster2-worker   <none>           <none>
+```
+
+##### 5. Export the Service on **cluster2**
+
+In order to signify that the Service should be visible and discoverable to other clusters in the cluster set, a `ServiceExport` needs to be
+created. The `subctl export` command can be used to automatically create the required `ServiceExport` object:
+
+```bash
+$ subctl export service --namespace nginx-test nginx
+Service exported successfully
+```
+
+##### 6. Consume the Service from **cluster3**
+
+First, verify that the exported `nginx` Service was imported to **cluster3** as expected. Submariner (via Lighthouse) automatically
+creates a corresponding `ServiceImport`:
+
+```bash
+$ kubectl config use-context cluster3
+Switched to context "cluster3".
+```
+
+```bash
+$ kubectl get -n submariner-operator serviceimport
+NAME                        TYPE           IP               AGE
+nginx-nginx-test-cluster2   ClusterSetIP   [100.2.29.136]   62s
+```
```

Comment:
I don't think we need this part here as the ServiceImport would already have been created from the initial export on cluster3.

Reply:
Indeed!